.buildkite/notebook-parameters.yml

```diff
@@ -1,7 +1,18 @@
 ---
 # configure the parameters replaced in jupyter notebooks when running on CI with papermill
-epochs: 2
-walk_length: 5
-batch_size: 5
+# common parameters used in several notebooks:
+epochs: 1
+batch_size: 2
+# short random walks
+walk_length: 3
+# minimal ensembles:
 n_estimators: 2
 n_predictions: 2
+# reduce graph size for movielens-recommender.ipynb and calibration-pubmed-link-prediction.ipynb:
+train_size: 0.005
+test_size: 0.01
+val_size: 0.01
+# minimal metapath for demos/embeddings/stellargraph-metapath2vec.ipynb:
+metapaths:
+  - - "group"
```

Comment:
Is `group`, `group` a metapath that exists in the graph? Maybe `group`, `user` would be more realistic? Also, by listing only one metapath, we're not validating that it correctly handles multiple. Is there a big slowdown by adding another short one?

Also, totally minor, but the double nesting of lists here was hard for me to understand, so maybe the inner one could use an inline list, e.g.

```yaml
metapaths:
  - ["group", "user"]
  - ["user", "user"]
```

Reply:
group-group gives just 47 paths, others give thousands and are much too slow (I'll confirm with a CI run though).

Reply:
confirmed it fails out after 12.5 minutes with a cell timeout adding user-user path (it times out doing the walks).

Reply:
have changed so that there are now two short metapaths for slightly better coverage: [["group", "group"], ["group", "user", "user", "group"]]

Reply:
👍 

> I feel that a minimal test here is better than no test.

Yeah, totally.